tests/ui/pages/management/tags-page.ts

```diff
@@ -1,5 +1,86 @@
 import ManagementPage from './management-page';
+import Button from "../../elements/button";
+import List from "../../elements/list";
+import DeleteTagModal from "../modals/delete-tag-modal";
 
-const SELECTORS = {};
+const SELECTORS = {
+    create_tag: `button:has-text("Create Tag")`,
+    tag_line_root: 'div[class*="MuiGrid-root MuiGrid-container css-1d3bbye"]',
+    tag_line_item: 'div[class*="MuiGrid-root MuiGrid-container sc-jOhDuK kJYgmA css-1d3bbye"]',
+    tag_line_item_name: '.MuiTypography-root.MuiTypography-body1.MuiTypography-noWrap.css-i5qci1',
```

Comment:
I think we need to update these selectors to make them clean and human-readable.
Using such long selectors will make tests fragile

Reply:
done